deploy/cloud/manifests/free-license.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: infostream.sealos.io/v1
```

Comment:
add `// nosemgrep: generic.secrets.security.detected-jwt-token.detected-jwt-token` to ingore semgrep ci .